app/src/main/java/com/kryptgames/health/fitwithfriends/CreateProfile.java

```diff
@@ -0,0 +1,326 @@
+                gallery.setAction(Intent.ACTION_GET_CONTENT);
+                startActivityForResult(Intent.createChooser(gallery, "Select Picture"), PICK_IMAGE);
+            }
+
+
+        });
+
+        databaseReference = FirebaseDatabase.getInstance().getReference("Profile");
+
+
+        Query myTopPostsQuery = FirebaseDatabase.getInstance().getReference("Profile");
+
+
+        myTopPostsQuery.addValueEventListener(new ValueEventListener() {
+            @Override
+            public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
+                Log.e("Count ", "" + dataSnapshot.getChildrenCount());
+                for (DataSnapshot postSnapshot : dataSnapshot.getChildren()) {
+
+
+                    Profile profile = postSnapshot.getValue(Profile.class);
+                    Log.e("Get Data", profile.getName());
+                    Log.e("Get Data", profile.getGenre());
+                    Log.e("Get Data", profile.getDob());
+                    Log.e("Get Data", profile.getHeight());
+                    Log.e("Get Data", profile.getWeight());
+
+
+                }
+            }
+
+            @Override
+            public void onCancelled(@NonNull DatabaseError databaseError) {
+
+            }
+        });
+
+
+        mDisplayDate = (TextView) findViewById(R.id.mDisplayDate);
+
+        edName = findViewById(R.id.edName);
+        edLastName = findViewById(R.id.edLastName);
+
+        edName.setFilters(new InputFilter[] {
+                new InputFilter() {
+                    public CharSequence filter(CharSequence src, int start,
+                                               int end, Spanned dst, int dstart, int dend) {
+                        if(src.equals("")){ // for backspace
+                            return src;
+                        }
+                        if(src.toString().matches("[a-zA-Z ]+")){
+                            return src;
+                        }
+                        return "";
+                    }
+                }
+        });
+
+
+        edLastName.setFilters(new InputFilter[] {
+                new InputFilter() {
+                    public CharSequence filter(CharSequence src, int start,
+                                               int end, Spanned dst, int dstart, int dend) {
+                        if(src.equals("")){ // for backspace
+                            return src;
+                        }
+                        if(src.toString().matches("[a-zA-Z ]+")){
+                            return src;
+                        }
+                        return "";
+                    }
+                }
+        });
+
+
+
+
+
+        spinnerGender = findViewById(R.id.spinnerGender);
+
+        save = findViewById(R.id.save);
+        weightSpinner = findViewById(R.id.weightSpinner);
+
+        final List<String> weight = new ArrayList<String>();
+        weight.add(0, "Weight");
+
+
+        for (int i = 70; i < 200; i++) {
```

Comment:
I think it is easily possible for someone to weight below 70 :) start from like 20